docs/DESIGN.md

```diff
@@ -21,30 +21,28 @@ available in `rust-vmm`:
 [`event-manager`](https://crates.io/crates/event-manager),
 [`vm-superio`](https://crates.io/crates/vm-superio) and
 [`vmm-sys-util`](https://crates.io/crates/vmm-sys-util). Only the serial
-console device will be supported. The next iteration will add virtio devices
-and vCPU abstractions. Without the latter, vCPU configuration will be handled
-directly through `kvm-ioctls` and will not be abstracted through interfaces.
+console device is supported. The current iteration is adding virtio devices
+and vCPU abstractions.
 
 Complex functionalities that require ongoing user interaction with the VMM are
-not supported in this version. Such functionalities include but are not
-limited to: device hotplug, VM pause/resume, snapshotting, migration. As the
-necessary building blocks emerge in `rust-vmm`, integration in the `vmm` crate
-will not pose a large effort, but the frontend / app will require extensive
-modifications.
+not currently supported. Such functionalities include but are not limited to:
+device hotplug, VM pause/resume, snapshotting, migration. We plan on adding the
+VM pause/resume support in the near future. As the necessary building blocks
```

Comment:
I've created an issue: https://github.com/rust-vmm/vmm-reference/issues/58. Can you mention the issue instead? Saying "near future" is a bit odd IMHO.

Reply:
Yup, done